### PR TITLE
Improve UI/UX for user page

### PR DESCRIPTION
**Pull request by azaria-dedmon**
**What does this PR do?**

- This PR makes improvements to the user homepage.

**Description of task to be completed.**

- Improve symmetry of elements on page.
- Remove unnecessary links
- Add new link to navbar.
- Improve search bar.

**How should this be manually tested?**

- This does not have to be tested.

### CODE REVIEW

**Comment by alexkayabula** [comment]
@azaria-dedmon change the title to `Improve UI/UX for the user page`

**Comment by alexkayabula** [comment]
The font used int the user page should be the same as the one used in the homepage.

**Comment by alexkayabula** [comment]
The navbar is missing a link or icon to the homepage.

**Comment by alexkayabula** [comment]
The navbar link `review testing site` should be `review testing sites`.

**Comment by azaria-dedmon** [comment]
@alexkayabula For the homepage, I used Bootstrap's Jumbotron div. Therefore I am not certain about the type of font that was used. However, I've compared the two pages to one another and they look the exact same to me. I cannot tell the difference.

File: app/templates/location.html [hunk 1]
```diff
@@ -18,11 +18,13 @@
             {% if not g.user %}
             <li><a href="/login">Login</a></li>
             <li><a href=“/location”>Testing Site Reviews</a></li>
-            <li><a href="/">Home</a></li>
+            <li><a href="/">Homepage</a></li>
             {% else %}
             <li><a href="/logout">Logout</a></li>
-            <li><a href=“/location”>Testing Site Reviews</a></li>
-            <li><a href="/user">Home</a></li>
+            <li><a href="/">Homepage</a></li>
```
Comment (alexkayabula) [inline app/templates/location.html]: @azaria-dedmon rename the Homepage link to Home and it should appear before the My Dashboard link in the navbar.

File: app/templates/base.html [hunk 2]
```diff
@@ -17,14 +17,16 @@
       {% else %}
       <form action="/search-user" method="get" id="search-form">
         <input class="search" type="text" name="username" placeholder="Search username">
-        <input class="search" type="submit" value="search">
+        <input id="search-btn"class="search" type="submit" value="search">
       </form>
       <li><a href="/logout">Logout</a></li>
-      <li><a href="/add/location/review">Testing Site Reviews</a></li>
-      <li><a href="/user">Home</a></li>
+      <li><a href="/">Homepage</a></li>
```
Comment (alexkayabula) [inline app/templates/base.html]: @azaria-dedmon rename the Homepage link to Home and it should appear before the My Dashboard link in the navbar.

File: app/templates/users/user_edit.html [hunk 3]
```diff
@@ -13,8 +13,10 @@
         <ul>
             <input type="text" placeholder="Search users">
             <li><a href="/logout">Logout</a></li>
-            <li><a href=“/location”>Testing Site Reviews</a></li>
-            <li><a href="/user">Home</a></li>
+            <li><a href="/">Homepage</a></li>
+            <li><a href=“/location”>Review Testing Sites</a></li>
+            <li><a href="/user/reviews">User Reviews</a></li>
+            <li><a href="/user">My Dashboard</a></li>
```
Comment (alexkayabula) [inline app/templates/users/user_edit.html]: This should be under a different PR. We are improving the UI/UX of the user page.

**Review by alexkayabula — CHANGES_REQUESTED** [review]

**Comment by alexkayabula** [comment]
> @alexkayabula For the homepage, I used Bootstrap's Jumbotron div. Therefore I am not certain about the type of font that was used. However, I've compared the two pages to one another and they look the exact same to me. I cannot tell the difference.

You can also use Bootstraps Jumbotron for the userpage so that the UI looks consistent.

File: app/templates/base.html [hunk 4]
```diff
@@ -17,14 +17,16 @@
       {% else %}
       <form action="/search-user" method="get" id="search-form">
         <input class="search" type="text" name="username" placeholder="Search username">
```
Comment (alexkayabula) [inline app/templates/base.html]: Change the Place holder to "Enter Username"

File: app/templates/base.html [hunk 5]
```diff
@@ -17,14 +17,16 @@
...
-        <input class="search" type="submit" value="search">
+        <input id="search-btn"class="search" type="submit" value="search">
```
Comment (alexkayabula) [inline app/templates/base.html]: Change the value to "Search" i.e capitalize the word

**Comment by azaria-dedmon** [comment]
> > @alexkayabula For the homepage, I used Bootstrap's Jumbotron div. Therefore I am not certain about the type of font that was used. However, I've compared the two pages to one another and they look the exact same to me. I cannot tell the difference.
> 
> You can also use Bootstraps Jumbotron for the userpage so that the UI looks consistent.

I cannot use Bootstraps "Jumbotron" feature as a small div. It is a hero styled banner; it's not suppose to be small in width. If I were to use it for the bottom two divs, they would both be as wide as the top banner.

**Comment by azaria-dedmon** [comment]
@alexkayabula Please disregard the previous comment.

**Comment by alexkayabula** [comment]
@azaria-dedmon I am going to look into this later on in the day, I am occupied at the moment. I will get back to you as soon as I am done reviewing. Thanx

**Comment by azaria-dedmon** [comment]
> @azaria-dedmon I am going to look into this later on in the day, I am occupied at the moment. I will get back to you as soon as I am done reviewing. Thanx

Okay. Thanks for the update.

**Review by alexkayabula — COMMENTED** [review]
@azaria-dedmon I have made some improvements to this PR. Please review, test out the changes and confirm then I can merge.

**Comment by azaria-dedmon** [comment]
> @azaria-dedmon I have made some improvements to this PR. Please review, test out the changes and confirm then I can merge.

I made a mistake and did not merge the commits that you made. I liked the changes, but decided to switch the default photo for another pic. If you could, please push those changes back to the branch. Sorry for the inconvenience.

**Pull request closed without merging**